### PR TITLE
global: cookiecutter template removal

### PR DESCRIPTION
**Pull request by lnielsen**
* Removes cookiecutter template in favor of standalone template.

### CODE REVIEW

**Comment by lnielsen** [comment]
This means that ``invenio instance create`` will be deprecated. Having the cookiecutter template integrated in the module was originally done so that the template would be in sync with the Invenio version, but every time we have an update to the template, we need a new invenio-base release, so I think it would be better to move the template out of the module and into a separate repo (where we can keep branches specific to specific invenio versions). @jma and @inveniosoftware/architects WDYT?

**Comment by lnielsen** [comment]
@krzysztof Check cookiecutter

**Comment by krzysztof** [comment]
@lnielsen the 'examples' are added in cookiecutter template: https://github.com/inveniosoftware/cookiecutter-invenio-module/blob/master/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/docs/conf.py#L11-L12

**Comment by krzysztof** [comment]
I haven't checked example app, but perhaps for this specific module we should have something in the docs about it?

File: .editorconfig [hunk 1]
```diff
@@ -32,7 +32,7 @@ charset = utf-8
 [*.py]
 indent_size = 4
 # isort plugin configuration
-skip = invenio_base/cookiecutter-invenio-base
+skip = .eggs invenio_base/cookiecutter-invenio-base
```
Comment (krzysztof) [inline .editorconfig]: remove ``invenio_base/cookiecutter-invenio-base`` ?

**Comment by lnielsen** [comment]
@krzysztof 👍 Please have another look. Regarding the MANIFEST then I think both options are fine. Important is that check-manifest is run to update it and that it's sorted to minimize potential merge conflicts.

**Comment by krzysztof** [comment]
@lnielsen I made some changes, take a look: https://github.com/inveniosoftware/invenio-base/pull/119/commits/58cac7de47b68e6c95b1aa86aee7fb86d767787f

I've unified the descriptions, but perhaps you wanted "Invenio application loader."?

**Comment by lnielsen** [comment]
Thanks a lot

**Pull request merged**